.github/workflows/tests.yaml

```diff
@@ -26,14 +25,23 @@ jobs:
       - name: "Checkout Repository"
         uses: actions/checkout@v4
 
-      - name: Set up Python
+      - name: Set up Python (Linux)
+        if: matrix.os == 'ubuntu-latest'
         uses: conda-incubator/setup-miniconda@v2
         with:
           mamba-version: "*"
```

Comment:
@trallard is removing this what you had in mind? It seems to work. The install on CI is a few minutes slower but I'm not sure if it's significant enough to worry about.

Reply:
I think this is doing the opposite, using conda only and not using mamba at all. But like I said, it's not *that* much slower.